zerver/views/video_calls.py

```diff
@@ -202,7 +202,8 @@ def get_bigbluebutton_url(
         {
             "meeting_id": id,
             "name": meeting_name,
-            "password": password,
+            "lock_settings_disable_cam": voice_only,
+            "moderator": request.user.get_username(),
```

Comment:
Are usernames now stable (as in they do not change when the mail/mail privacy is changed?) or do we want to use the ID instead?

Reply:
Thanks for pointing this out! IDs are definitely more fool proof so I've changed the code to use them instead